src/app/unit-host/unit-host.component.spec.ts

```diff
@@ -25,7 +24,7 @@ describe('UnitHostComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [RouterTestingModule, MatTooltipModule],
+      imports: [RouterTestingModule, AppModule],
```

Comment:
Warum hier AppModule statt dem konkreten Material-Modul?

Reply:
Es geht weniger um das MatTooltipModule, als vielmehr um die neu eingeführte DenyNavigationComponent, die im AppModule definiert ist.

Reply:
gelöst in [in diesem commit](https://github.com/iqb-berlin/verona-player-testbed/pull/14/commits/c222aa9eb073d02154fb12cafb4b0bc12b2c2df5)